### PR TITLE
Hashmap repeated word

### PR DESCRIPTION
**Pull request by Nicholas-Mercado**
- [ x] Top-level README “Table of Contents” is updated
 - [x ] README for this challenge is complete
       - [x ] Summary, Description, Approach & Efficiency, Solution
       - [ x] Picture of whiteboard
       - [ x] Link to code
 - [ x] Feature tasks for this challenge are completed
 - [x ] Unit tests written and passing
       - [ x] “Happy Path” - Expected outcome
       - [x ] Expected failure
       - [ x] Edge Case (if applicable/obvious)

### CODE REVIEW

File: python/code_challenges/hashtable_repeated_word.py [hunk 1]
```diff
@@ -1,5 +1,28 @@
...
+    lower_string = re.sub(r'\n', '', lower_string)
+    keys = lower_string.split(' ')
+    stripped_words = []
+
+    # remove special characters from each list item
+    for word in keys:
+        text = re.sub(r"[!?',;.]", '', word)
+        stripped_words.append(text)
+    # Create dict
+    word_dict = dict.fromkeys(stripped_words, 0)
```
Comment (chloenott) [inline python/code_challenges/hashtable_repeated_word.py:17]: This is cool, creating a dictionary with fromkeys!

File: python/docs/hashtable_repeated_word/README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+![Whiteboard](hashmap_repeated_word.JPG)
+
+## Approach & Efficiency
+
+This algorithms approach is to split the the string into a list and create a dictionary from each word. It then iterates through that original list incrementing each dictionary key as it is found. If any dictionary keys value reaches two it returns that key.
+
+## Big 0
+
+Time Complexity --> O(n) because worst case if there are no matching words it must iterate through the whole string.
```
Comment (chloenott) [inline python/docs/hashtable_repeated_word/README.md:15]: Time complexity looks correct and with the right justification. The whiteboard mentions space complexity as O(n) as well which I believe is correct. Recommend including its justification here as well.

**Review by chloenott — COMMENTED** [review]
Good work! Well-organized PR and commits! Glad you squeezed in a code cleanup refactor.
